### PR TITLE
Bug 1950532: Include "update" when referring to operator approval

### PR DESCRIPTION
**Pull request by florkbr**
Updates the install operator page terminology to use update approval and
aligns the installed operators subscription tab to use the same help icon and
text for update channel and update approval.

Screen Shot 2021-04-19 at 4 20 19 PM
Screen Shot 2021-04-19 at 4 20 35 PM

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1950532

### CODE REVIEW

**Comment by spadgett** [comment]
cc @ahardin-rh in case this has doc impact. (I know we walk through installing operators in several places.)

File: frontend/packages/operator-lifecycle-manager/locales/en/olm.json [hunk 1]
```diff
@@ -299,6 +299,7 @@
   "Remove Subscription": "Remove Subscription",
   "View ClusterServiceVersion...": "View ClusterServiceVersion...",
   "Channel": "Channel",
+  "Approval strategy": "Approval strategy",
```
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/locales/en/olm.json:302]: Hm, I'm surprised this is still in the messages. It looks like you got all the occurrences I could find.

```
❯ rg 'Approval strategy' -tts -tjs
packages/operator-lifecycle-manager/src/components/subscription.tsx
265:        title: t('olm~Approval strategy'),

packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx
732:                  <label className="co-required">{t('olm~Approval strategy')}</label>

packages/operator-lifecycle-manager/src/components/subscription.spec.tsx
296:      'olm~Approval strategy',
```

@florkbr Did you run `yarn i18n`?

Comment (florkbr) [reply frontend/packages/operator-lifecycle-manager/locales/en/olm.json:302]: @spadgett I did run `yarn i18n`. Note there is still one place I think the string is used here: https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/subscription.tsx#L265

I'm not sure if that needs to be updated as well. I could not find where it is rendered in the UI.

Comment (spadgett) [reply frontend/packages/operator-lifecycle-manager/locales/en/olm.json:302]: Ah ha, you can see that on the Search page here:

http://localhost:9000/search/ns/openshift-operators?kind=operators.coreos.com%7Ev1alpha1%7ESubscription

Screen Shot 2021-04-20 at 3 16 47 PM

I'd update that instance as well so we're consistent.

Comment (florkbr) [reply frontend/packages/operator-lifecycle-manager/locales/en/olm.json]: @spadgett done!

Comment (florkbr) [reply frontend/packages/operator-lifecycle-manager/locales/en/olm.json]: Screen Shot 2021-04-21 at 11 01 37 AM

**Review by spadgett — APPROVED** [review]
/lgtm

👍

**Pull request merged**